src/components/Lend/AaveAction.tsx

```diff
@@ -0,0 +1,46 @@
+import { TxBuilderV2, Network, Market } from "@aave/protocol-js";
```

Comment:
This file can just be a `ts` file

Reply:
what would that look like? this is one of the installed packages

Reply:
sorry i don't know how to make file comments on git (was also using the ios app when making the previous comment).


I meant `AaveAction.tsx` can just be `AaveAction.ts` you only use `jsx` or `tsx` when it's using React HTML/JS.